src/main/java/no/sikt/sws/IndexHandler.java

```diff
@@ -34,7 +34,10 @@ protected ProxyResponse<String> processProxyInput(
 
         var resourceIdentifier = request.getPathParameter(RESOURCE_IDENTIFIER);
 
-        if (resourceIdentifier.startsWith("_")) {
+        // Regex: may only contain big and small letters, digits, '/', '?', '&', '-' and '_'
+        var regex = "[A-Za-z\\d/?&_-]*";
```

Comment:
Should we extract this as a constant?
Should we allow norwegian chars like ÆØÅ?